CharlieBackend.Business/Services/AccountService.cs

```diff
@@ -80,17 +81,17 @@ public async Task<AccountDto> GetAccountCredentialsAsync(AuthenticationDto authe
 
                 var foundAccount = _mapper.Map<AccountDto>(await _unitOfWork.AccountRepository.GetAccountCredentials(authenticationModel));
 
-                return foundAccount;
+                return Result<AccountDto>.GetSuccess(foundAccount);
             }
 
             return null;
```

Comment:
Create a return here for Result<T>.Error() instead of return null

Reply:
Which error should I return?
NotFound?

Reply:
It depends why we return null